src/Docker.DotNet.BasicAuth/Docker.DotNet.BasicAuth.csproj

```diff
@@ -7,4 +7,10 @@
   <ItemGroup>
     <ProjectReference Include="..\Docker.DotNet\Docker.DotNet.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.3.37">
```

Comment:
What is this for? We shouldnt need this

Reply:
Found this reference to this package
https://github.com/dotnet/Docker.DotNet/commit/bbe5d8f905568436595e3b0df5537657e35171cd#diff-5c1900dca99df01c3ddc3a884282164b04deee76e79b176e2d34cfa47f3b8907

I can't build current master branch  without adding these references. I do not know what are these for

Reply:
Oh wow. I dont know how this went in. @galvesribeiro - Did you add this? We should remove this dependency IMO but not this PR's fault.

Reply:
This can't be removed. 

It is used to stamp the assemblies with a deterministic version based on the git commit. It is only of the requirements to have the assembly signed by the sign tools which is a requirement to be on .Net Foundation. 

@dgvives that was the PR I added the sign tools.